index.html

```diff
@@ -64,7 +64,9 @@ <h1>Save the Date</h1>
             to let us know whether or not you can attend, so we can reserve and make seating arrangements for the event.
         </p>
         <p>
-          Add to Calendar
+          Add to Calendar <br> 
+          <a target="_blank" href="https://calendar.google.com/event?action=TEMPLATE&amp;tmeid=M2pqc3U3aHN2NzRvdW1lYWxjNjJnYzcza3EgYW5pdGE4MjU5M0Bt&amp;tmsrc=anita82593%40gmail.com"><img border="0" align="center" alt="Google Calendar Logo" height=60 src="https://46p2yg36reid1zp9ic1aj732-wpengine.netdna-ssl.com/wp-content/uploads/2014/02/gcallogo2.png"></a>
```

Comment:
The img src should not be external, just in case the source goes down. Download the image, add it to this commit, then use a relative path to it. For an example of a relative path, look at the header image.